packages/tools/replay-tool/src/replayFluidFactories.ts

```diff
@@ -31,7 +35,8 @@ import {
 export class ReplayRuntimeFactory extends RuntimeFactoryHelper {
     constructor(
         private readonly runtimeOptions: IContainerRuntimeOptions,
-        private readonly registries: NamedFluidDataStoreRegistryEntries) {
+        private readonly registries: NamedFluidDataStoreRegistryEntries,
+        private readonly requestHandlers: RuntimeRequestHandler[] = []) {
```

Comment:
I think we should add the `containerRuntimeResolver` here by default rather than this being passed. All current usages need it so this doesn't have to be optional.
Or, at least have it by default in `loadContainer` method.

Reply:
You could also accept it as a param but have `containerRuntimeResolver` as default anyway.

Reply:
Yes - good callout. Since we depend on it, let's just make sure it's loaded regardless if other handlers are present or not.